### ISSUE TITLE
webpack: npm run build

### ISSUE THREAD

**Issue by fundon** [fundon/vue-admin#39]
```
/Users/fundon/Dev/repos/vue-admin/node_modules/loader-runner/lib/LoaderRunner.js:90
			throw e;
			^

TypeError: Cannot read property '/Users/fundon/Dev/repos/vue-admin/node_modules/extract-text-webpack-plugin' of null
    at /Users/fundon/Dev/repos/vue-admin/node_modules/extract-text-webpack-plugin/index.js:257:27
    at /Users/fundon/Dev/repos/vue-admin/node_modules/webpack/lib/Compilation.js:462:4
    at Array.forEach (native)
    at callback (/Users/fundon/Dev/repos/vue-admin/node_modules/webpack/lib/Compilation.js:461:14)
    at Compilation.<anonymous> (/Users/fundon/Dev/repos/vue-admin/node_modules/webpack/lib/Compilation.js:482:4)
    at /Users/fundon/Dev/repos/vue-admin/node_modules/webpack/lib/Compilation.js:326:10
    at /Users/fundon/Dev/repos/vue-admin/node_modules/async/lib/async.js:52:16
    at Object.async.forEachOf.async.eachOf (/Users/fundon/Dev/repos/vue-admin/node_modules/async/lib/async.js:236:30)
    at Object.async.forEach.async.each (/Users/fundon/Dev/repos/vue-admin/node_modules/async/lib/async.js:209:22)
    at Compilation.addModuleDependencies (/Users/fundon/Dev/repos/vue-admin/node_modules/webpack/lib/Compilation.js:192:8)
    at Compilation.processModuleDependencies (/Users/fundon/Dev/repos/vue-admin/node_modules/webpack/lib/Compilation.js:177:7)
    at Compilation.<anonymous> (/Users/fundon/Dev/repos/vue-admin/node_modules/webpack/lib/Compilation.js:469:8)
    at /Users/fundon/Dev/repos/vue-admin/node_modules/webpack/lib/Compilation.js:124:4
    at Array.forEach (native)
    at callback (/Users/fundon/Dev/repos/vue-admin/node_modules/webpack/lib/Compilation.js:123:12)
    at /Users/fundon/Dev/repos/vue-admin/node_modules/webpack/lib/Compilation.js:147:10
    at setError (/Users/fundon/Dev/repos/vue-admin/node_modules/webpack/lib/NormalModule.js:214:3)
    at /Users/fundon/Dev/repos/vue-admin/node_modules/webpack/lib/NormalModule.js:178:18
    at /Users/fundon/Dev/repos/vue-admin/node_modules/webpack/lib/NormalModule.js:137:11
    at /Users/fundon/Dev/repos/vue-admin/node_modules/loader-runner/lib/LoaderRunner.js:328:11
    at /Users/fundon/Dev/repos/vue-admin/node_modules/loader-runner/lib/LoaderRunner.js:150:20
    at context.callback (/Users/fundon/Dev/repos/vue-admin/node_modules/loader-runner/lib/LoaderRunner.js:87:13)
    at Object.<anonymous> (/Users/fundon/Dev/repos/vue-admin/node_modules/extract-text-webpack-plugin/loader.js:116:12)
    at Compiler.<anonymous> (/Users/fundon/Dev/repos/vue-admin/node_modules/webpack/lib/Compiler.js:251:10)
    at /Users/fundon/Dev/repos/vue-admin/node_modules/webpack/lib/Compiler.js:442:12
    at next (/Users/fundon/Dev/repos/vue-admin/node_modules/tapable/lib/Tapable.js:81:11)
    at Compiler.<anonymous> (/Users/fundon/Dev/repos/vue-admin/node_modules/extract-text-webpack-plugin/loader.js:82:4)
    at next (/Users/fundon/Dev/repos/vue-admin/node_modules/tapable/lib/Tapable.js:83:14)
    at Compiler.<anonymous> (/Users/fundon/Dev/repos/vue-admin/node_modules/webpack/lib/CachePlugin.js:61:4)
    at Compiler.applyPluginsAsync (/Users/fundon/Dev/repos/vue-admin/node_modules/tapable/lib/Tapable.js:85:13)

npm ERR! Darwin 16.0.0
npm ERR! argv "/Users/fundon/.nvm/versions/node/v6.3.1/bin/node" "/Users/fundon/.nvm/versions/node/v6.3.1/bin/npm" "run" "build"
npm ERR! node v6.3.1
npm ERR! npm  v3.10.5
npm ERR! code ELIFECYCLE
npm ERR! vue-admin@0.0.1 build: `rm -rf dist && webpack --config build/webpack.prod.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vue-admin@0.0.1 build script 'rm -rf dist && webpack --config build/webpack.prod.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the vue-admin package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rm -rf dist && webpack --config build/webpack.prod.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs vue-admin
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls vue-admin
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/fundon/Dev/repos/vue-admin/npm-debug.log
```

**Comment by fundon** [comment]
Related to: https://github.com/webpack/extract-text-webpack-plugin/issues/226

**Comment by tudousi991112** [comment]
+1

**Comment by fundon** [comment]
Back to Webpack v1.

**Comment by fundon** [comment]
Landed in f6c90bc.
